docs/reference/UDS Core/functional-layers.md

```diff
@@ -46,3 +46,41 @@ packages:
     repository: ghcr.io/defenseunicorns/packages/uds/core-backup-restore
     ref: 0.29.0-upstream
 ```
+
+## Layer Selection
+
+Layer selection will always be deployment-specific but below are guidelines for what layers to consider for your deployment.  The layers marked with a four pointed star (🟄) are those needed to follow the [Big Bang Conformant Stack](https://repo1.dso.mil/big-bang/product/bbtoc/-/blob/master/policy/conformance.md?ref_type=heads) though if you are not bound by that document, UDS Core Base is the only *technical* layer required to install most UDS Packages.
+
+| Layer | Selection Criteria |
+|-------|--------------------|
+| **UDS Add-ons*** |
+| UDS UI           | Provides a common operating picture for a Kubernetes cluster and UDS deployments <br/> *(install if you would like to have an easy-to-use window into your cluster/deployments)* |
+| UDS Registry     | Provides a storage location for UDS components and mission applications <br/> *(install if you would like to be able to easily store and view the software available in your environment)* |
+| UDS Remote Agent | Allows for more advanced remote cluster management / deployment <br/> *(install if you would like to manage UDS deployments from more advanced clients than UDS CLI)* |
+| **UDS Core Layers** |
+| Runtime Security†🟄         | Provides more advanced security with runtime inspection <br/> *(install if resources allow and more advanced security is desired)* |
+| Monitoring†                | Provides frontend log / metrics monitoring and alerting <br/> *(install if resources allow and more advanced debugging is desired)* |
+| Backup and Restore         | Allows volumes and k8s objects to be backed up and restored <br/> *(install if deployment provides critical data or must maintain state)* |
+| Identity and Authorization | Provides authentication and authorization functionality <br/>*(install if deployment requires an auth mechanism (i.e. direct user login))*  |
+| Logging🟄                   | Provides backend log storage and log shipping capabilities <br/> *(install if the deployment requires log aggregation and shipping)* |
+| Metrics Server††🟄          | Provides metrics collection capabilities (req of UDS Runtime) <br/> *(install if the cluster does not provide its own metrics server)* |
+| Base🟄                      | Provides the base for all other functional layers <br/> *(required for all "UDS" deployments and all other functional layers)* |
+| **UDS Core Pre-Requisites**‡ |
+| UDS Package Minio Operator | Provides storage for Loki (Logging) and Velero (Backup and Restore) <br/> *(install after core base but before logging/backup and restore if selected)* |
```

Comment:
2 nits:
- might leave off the `loki` and `velero` names since we don't refer to those in the above layers (or at least invert so the layer name is first and app is in parenthesis)
- would we want to note/encourage someone to use minio for backup storage? I'm not super familiar with what this might look like on-prem/on-edge, would it be normal to backup to minio and then backup minio itself somewhere else?